OpenEdXMobile/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -89,7 +85,7 @@ public void onClick(View v) {
                 if (NetworkUtil.isConnected(LoginActivity.this)) {
                     showResetPasswordDialog();
                 } else {
-                    showNoNetworkDialog();
+                    showAlertDialog(getString(R.string.reset_no_network_title), getString(R.string.network_not_connected));
```

Comment:
I would consider that out of scope here, cuz, all these error messages' names are defined in a pattern and all of them should be refactored in one go.